src/util/DeleteClassService.js

```diff
@@ -0,0 +1,45 @@
+import { deleteClass, getUserData } from './DataBaseRequests';
+
+import { useAuth } from '../AuthProvider';
+import { useState } from 'react';
+
+const useDeleteClass = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { userData, setUserData } = useAuth();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const openModal = () => setIsModalOpen(true);
+  const closeModal = () => setIsModalOpen(false);
+
+  const handleDelete = async (token, classId, onRequestClose, onError) => {
```

Comment:
If I have only 1 class and delete it receive the info about the class that i hav just deleted> but i have to get a page with with button create your first class

Reply:
got it. I'll fix it.

Reply:
fixed it.